tileview/src/main/java/com/qozix/tileview/widgets/ZoomPanLayout.java

```diff
@@ -911,4 +934,22 @@ public float getInterpolation( float input ) {
     void onZoomEnd( float scale, Origination origin );
   }
 
+  public enum MinimumScaleMode {
+    /**
+     * Limit the minimum scale to no less than what
+     * would be required to fill the container
+     */
+    FILL,
+
+    /**
+     * Limit the minimum scale to no less than what
+     * would be required to fit inside the container
+     */
+    FIT,
```

Comment:
I don't see any reference to that case in the code. But i see where in `calculateMinimumScaleToFit`, setting `MinimumScaleMode` to `FIT` would have an effect.